packages/release/release.spec

```
@@ -175,12 +182,13 @@ Requires: (%{name}-fips if %{_cross_os}image-feature(fips))
 %install
 install -d %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}
 install -p -m 0644 %{S:11} %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}
+# Add art to the console
+install -p -m 0644 %{S:1107} %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/issue
```

after renumbering the source, you can merge with the previous line and drop the comment:
```suggestion
install -p -m 0644 %{S:11} %{S:12} %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}
```